app/MyApp/app/collections/Members.js

```diff
@@ -5,7 +5,9 @@ $(function() {
 		url: function() {
 			if (this.login) {
 				return App.Server + '/members/_design/bell/_view/MembersByLogin?include_docs=true&key="' + this.login + '"'
-			} else if (this.skip) {
+			} else if (this.manager) {
+                return App.Server + '/members/_design/bell/_view/managerMembers?include_docs=true'
+            } else if (this.skip) {
```

Comment:
please make this go to the right indent

if you do that it will be the same as old code line 8